Dockerfile

```diff
@@ -1,11 +1,12 @@
 FROM praekeltfoundation/python-base
 
-ENV SUPERVISOR_VERSION "3.2.1"
+ENV SUPERVISOR_VERSION "3.2.3"
 RUN pip install supervisor==$SUPERVISOR_VERSION
 
 # Mimic the Debian/Ubuntu config file structure
-ADD ./supervisor.conf /etc/supervisor/supervisord.conf
-RUN mkdir -p /etc/supervisor/conf.d && \
+ADD ./supervisord.conf /etc/supervisord.conf
+RUN mkdir -p /etc/supervisord.d && \
+    mkdir -p /etc/supervisor/conf.d && \
     mkdir -p /var/log/supervisor
```

Comment:
You could put these all in the same command: `mkdir -p /etc/supervisord.d /etc/supervisor/conf.d /var/log/supervisor`